### ISSUE TITLE
[Suggestion] Different design for channels on search

### ISSUE THREAD

**Issue by okan35** [ram-on/SkyTube#678]
As the title states, when searching for videos channels are also listed but their design is same with videos. I even thought that something was wrong with the as I thought suddenly it was not showing channel names on videos. So in my opinion the design in original youtube is so good at this as it directly shows the difference between a video and a channel unlike Skytube.

**Comment by gzsombor** [comment]
Yes, it's a not easily distinguishable - or only in a bad way, as the picture size is a bit different, so the  row layout is a bit broken (that would like to fix before the next release). I haven't checked yet, how the official Youtube client display the channels - or maybe I've checked, but I forgot :disappointed:

**Comment by okan35** [comment]
This is difference between youtube vanced  and skytube
https://ibb.co/37PxfdJ 

As you see there is a distinct difference between a video and a channel. I will check this if I can

**Comment by okan35** [comment]
@gzombor What do you think about this ?



https://ibb.co/tXpCkxY

I actually wanted to put a margin of 20dp or something when video_cell is a channel but just couldn't get Constraint layout to i initialize correctly and change its margin. In my opinion a margin around video cell when it is a channel would be great. For now I just make the imageview smallar if it is a channel.

**Comment by gzsombor** [comment]
I fear, that changing the height of the cell is not the best idea, please test the app on tablet - when multiple columns are displayed. In this case, if the cells have different sizes, the whole row could be displayed strangely.

**Comment by okan35** [comment]
after looking atthet newpipe also. In my opinion the best is to load the a new layout instead of video_cell layout. It looks different now but still not the best. If I have time I will try tomake a new layout that looks a channel.
